bundles/org.openhab.core.addon/src/main/java/org/openhab/core/addon/internal/xml/AddonInfoAddonsXmlProvider.java

```diff
@@ -79,8 +79,18 @@ public Set<AddonInfo> getAddonInfos(@Nullable Locale locale) {
     }
 
     private void initialize() {
+        File fileFolder = new File(folder);
+        try {
+            if (!fileFolder.isDirectory()) {
+                logger.warn("Folder '{}' does not exist", folder);
```

Comment:
I would reduce this to debug - if the folder is not there, we simply do not provide any AddonInfos as none are present; but maybe that is intended for some setup, who knows.

Reply:
Done